index.js

```diff
@@ -191,26 +191,26 @@ function encode(err, buffer, headers) {
 
     if (err)  {
         headers['x-redis-err'] = errcode(err).toString();
-        headers = new Buffer(JSON.stringify(headers), 'utf8');
+        headers = Buffer.from(JSON.stringify(headers), 'utf8');
         return headers;
     }
 
     // Turn objects into JSON string buffers.
     if (buffer && typeof buffer === 'object' && !(buffer instanceof Buffer)) {
         headers['x-redis-json'] = true;
-        buffer = new Buffer(JSON.stringify(buffer));
+        buffer = Buffer.from(JSON.stringify(buffer));
     // Turn strings into buffers.
     } else if (buffer && !(buffer instanceof Buffer)) {
-        buffer = new Buffer(buffer);
+        buffer = Buffer.from(buffer);
     }
 
-    headers = new Buffer(JSON.stringify(headers), 'utf8');
+    headers = Buffer.from(JSON.stringify(headers), 'utf8');
 
     if (headers.length > 1024) {
         throw new Error('Invalid cache value - headers exceed 1024 bytes: ' + JSON.stringify(headers));
     }
 
-    var padding = new Buffer(1024 - headers.length);
+    var padding = Buffer.allocUnsafe(1024 - headers.length);
```

Comment:
Why not use: `Buffer.alloc(size[, fill[, encoding]])` and skip the `fill` on the next line?

Reply:
Done in https://github.com/mapbox/tilelive-redis/pull/38/commits/c470925203d6ad2eff412c934a7aaf5b91b35a7d

Reply:
👍